### PR TITLE
Remove unnecessary verbiage

### PR DESCRIPTION
**Pull request by dougbu**
- use `$(AspNetCoreMajorMinorVersion)` more
  - confirmed using `git show|sls '^- '|sls -NotMatch '\$\(AspNetCoreMajorVersion\)\.\$\(AspNetCoreMinorVersion\)'`

### CODE REVIEW

**Comment by dougbu** [comment]
Split this out of #24983

File: src/Components/WebAssembly/WebAssembly/src/Microsoft.AspNetCore.Components.WebAssembly.csproj [hunk 1]
```diff
@@ -22,7 +22,10 @@
       UndefineProperties="TargetFramework"
       Private="false"
       Condition="'$(BuildNodeJS)' != 'false' and '$(BuildingInsideVisualStudio)' != 'true'" />
+  </ItemGroup>
 
+  <ItemGroup Condition=" '$(AspNetCoreMajorMinorVersion)' == '6.0' ">
```
Comment (dougbu) [inline src/Components/WebAssembly/WebAssembly/src/Microsoft.AspNetCore.Components.WebAssembly.csproj:27]: Oops, should be `5.0` until #24983 gets in❕

Comment (dougbu) [reply src/Components/WebAssembly/WebAssembly/src/Microsoft.AspNetCore.Components.WebAssembly.csproj:27]: Or, perhaps these suppressions can be remove too -- no errors so far❔

**Comment by dougbu** [comment]
Ignore for now. Even if the old suppressions are wrong, looks like we need a lot.

**Comment by dougbu** [comment]
Removed everything about `@(SuppressBaselineReference)`. Work there is needed in `release/3.1` and `release/5.0` as we're not correctly checking against the baselines.

**Review by JunTaoLuo — APPROVED** [review]

**Pull request merged**